### PR TITLE
adds wagtail

### PR DESCRIPTION
**Pull request by brianedelman**

### CODE REVIEW

File: hooks/post_gen_project.sh [hunk 1]
```diff
@@ -41,6 +41,17 @@ fi
 
 {% endif %}
 
+
+{% if cookiecutter.use_wagtail == 'n' %}
+# Delete wagtail if not using
+echo "Deleting wagtail app";
```
Comment (gone) [inline hooks/post_gen_project.sh:47]: I want to figure out a better way to do this - One option would be including the files and just having them empty

File: {{cookiecutter.repo_name}}/requirements-dev.txt [hunk 2]
```diff
@@ -18,5 +18,5 @@ astroid==2.2.5
 prospector==1.1.7
 pylint==2.3.1
 service-factory==0.1.5
-Werkzeug==0.12.2
+Werkzeug==0.15.3
```
Comment (gone) [inline {{cookiecutter.repo_name}}/requirements-dev.txt:21]: I think this conflicts

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/blocks/column.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{%raw%}<div class="{{ class_name }}">
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/blocks/column.html:1]: I think we can use - to better strip whitespace, this is jinja right

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/blocks/link_block.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{%raw%}
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/blocks/link_block.html:1]: ditto here

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/wagtailapp/base_page.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{%raw%}{% extends 'base.jinja' %}
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/wagtailapp/base_page.html:1]: Isn't this an abstract page?

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/wagtailapp/tests.py [hunk 6]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/wagtailapp/tests.py:3]: Could we make some render tests for the the custom blocks here? Just to show how it's done

**Comment by brianedelman** [comment]
@gone Added more tests. They aren't perfect because wagtail streamblocks are hard to test. Added the wagtail option to the test file so we should be in good standing in terms of a build process.

File: cookiecutter.json [hunk 7]
```diff
@@ -2,12 +2,19 @@
...
     "django_registration": "y",
     "stripe": "y",
     "org_name": "Lightmatter",
     "description": "A short description of the project.",
     "domain_name": "example.com",
     "version": "0.1.0",
-    "use_local_docker_db": "n"
+    "use_local_docker_db": "n",
+    "use_wagtail": "n",
+    "_copy_without_render": [
```
Comment (gone) [inline cookiecutter.json:15]: So doing this will let us get away from raw tags right

**Pull request merged**